tests/io/mock_data/event000000000-cells.csv

```diff
@@ -1,11 +1,11 @@
 geometry_id,hit_id,channel0,channel1,timestamp,value
-504403295704449025,0,1,0,0,0.0041470062
-504403295704449025,0,0,1,0,0.00306466641
-504403295704449025,0,1,1,0,0.00868905429
-504403295704449025,1,1,1,0,0.00886478275
-504403295704449025,1,1,2,0,0.00580428448
-504403295704449025,1,2,1,0,0.0016894876
-504403295704449025,1,2,2,0,0.00199076766
-504403295704449025,2,5,5,0,0.00632160669
-504403295704449025,2,5,6,0,0.00911649223
-504403295704449025,2,5,7,0,0.00518329488
+576460889742407168,0,1,0,0,0.0041470062
+576460889742407168,0,0,1,0,0.00306466641
+576460889742407168,0,1,1,0,0.00868905429
+576460889742407168,1,1,1,0,0.00886478275
+576460889742407168,1,1,2,0,0.00580428448
+576460889742407168,1,2,1,0,0.0016894876
+576460889742407168,1,2,2,0,0.00199076766
+576460889742407168,2,5,5,0,0.00632160669
+576460889742407168,2,5,6,0,0.00911649223
+576460889742407168,2,5,7,0,0.00518329488
```

Comment:
Out of curiosity, did you do these updates by hand, or did you use my script from https://gitlab.cern.ch/acts/traccc-data/-/merge_requests/8? :smile:

Reply:
I did these by hand as it was just a single geometryid repeated over

Reply:
:+1: